install/tutorial_9_later.md

```diff
@@ -180,22 +180,22 @@ Miniconda suffices.
 
 Create if necessary, and activate a Conda environment:
 
-    conda create -n ign-ws
-    conda activate ign-ws
+    conda create -n gz-ws
+    conda activate gz-ws
 
 Install prerequisites:
 
     conda install urdfdom --channel conda-forge
 
-Install Ignition dependencies:
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libignition-math<#> libgz-tools<#> --channel conda-forge
```

Comment:
I think we should only use `libgz-*` packages for garden packages. Can you rever this and add a separate line for `libsdformat13` and later?